src/lib/cli/command.js

```diff
@@ -29,21 +28,17 @@ function uncaughtError( err ) {
 process.on( 'uncaughtException', uncaughtError );
 process.on( 'unhandledRejection', uncaughtError );
 
-async function initAnalytics() {
-	const token = await Token.get();
-	const uuid = await token.uuid();
-	return analytics( uuid );
-}
-
 let _opts = {};
 args.argv = async function( argv, cb ): Promise<any> {
-	const tracker = await initAnalytics();
-
 	const options = this.parse( argv );
 
 	const validationError = validateOpts( options );
 	if ( validationError ) {
-		console.log( validationError.toString() );
+		const error = validationError.toString();
+
+		trackEvent( 'command_validation_error', { error } );
```

Comment:
Is there any risk of process.exit() firing before this Promise resolves?

Reply:
Oh, good catch. We need `await` before all those calls.